doc/_static/version_switcher.json

```diff
@@ -0,0 +1,34 @@
+[
+    {
+        "version": "0.1.x",
+        "url": "https://jni.github.io/skan/0.1.x"
```

Comment:
not sure how the url's work so not sure if this works, pydata docs didn't expand. @jni can you explain..?

Reply:
Probably can replace this with `skeleton-analysis.org` instead of `jni.github.io/skan`. But I think both will work, because `jni.github.io/skan/whatever.html` automatically resolves to `skeleton-analysis.org/whatever.html`. In fact what you've got is probably the optimum choice because it'll be stable even if the domain lapses.

Reply:
For each of these we need to go back and build the old docs though, and the early versions are pointless. I would say let's go back to 0.7.x and no more, as that is the version matching the paper.

Reply:
I kept `jni.github.io/skan` because that is what is used everywhere else in the config file. So keep as is then?

Reply:
Yup!